source/support/zendesk/index.html.md.erb

```diff
@@ -262,29 +262,29 @@ You'll be given 2 options: To subscribe to email updates or to create an account
 
 Thank you for your message and sorry to hear you're having trouble with your account.
 
-You've reached the GOV.UK Account team - we are the team responsible for maintaining the GOV.UK Account. The GOV.UK Account is the one that you can currently use to save your answers from the Brexit checker - to help you work out what you need to do differently now that the UK has left the EU.
+You've reached the GOV.UK Account team - we are the team responsible for maintaining the GOV.UK Account. The GOV.UK Account is the one that you can currently use to save your answers from the [Brexit checker](https://www.gov.uk/transition-check/questions). The Brexit checker helps you work out what you need to do differently now that the UK has left the EU.
```

Comment:
This will be in an email so not sure we can have inline links. could try:

```suggestion
You've reached the GOV.UK Account team - we are the team responsible for maintaining the GOV.UK Account. The GOV.UK Account is the one that you can currently use to save your answers from the Brexit checker (https://www.gov.uk/transition-check/questions). The Brexit checker helps you work out what you need to do differently now that the UK has left the EU.
```